.github/workflows/snyk.yml

```diff
@@ -0,0 +1,19 @@
+name: Snyk
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  security:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+      
+    - name: Run Snyk to check for vulnerabilities
+      uses: snyk/actions/scala@master
+      with:
+        command: monitor
```

Comment:
```suggestion
        command: monitor
        args: --project-name=stream-collector --org=data-cap
```

Reply:
You don't need to specify --org=data-cap if the service account token is generated against the Data Cap organisation in Snyk

Reply:
Good, to know! Thanks @paulboocock!

Reply:
But somehow you do need to specify `--project-name` if I'm not mistaken.

Reply:
You'll probably need `args: --project-name=stream-collector` too. It tries to find the project name from config, or it defaults to the containing folder name. Unfortunately Github actions maps this into a `workspace` folder in the docker image that's running behind the scenes. So specifying `--project-name` just guarentees we get nice names in the Snyk UI.

Reply:
@benjben I think this is explained, could you change PR status, please?